tools/assemble_wat.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const { execFileSync } = require("child_process");
+
+process.argv[2].split();
+
+const path = process.argv[2].split("/");
+const filename = path.pop().split(".")[0];
+process.chdir(path.join("/"));
+
+execFileSync("wat2wasm", [filename + ".wat", "--enable-all"]);
```

Comment:
Myabe check `wat2wasm` exists before executing it, and show proper error messages if it doesn't exist?